nilearn/plotting/displays/_figures.py

```diff
@@ -425,29 +426,6 @@ def _get_sorted_edge_centroids(self, parc_idx, elevation=0.1):
 
         return np.asarray(sorted_vertices)
 
-    def _get_faces_on_edge(self, parc_idx):
-        """Identify which faces lie on the outeredge of the parcellation \
-        defined by the indices in parc_idx.
-
-        Parameters
-        ----------
-        parc_idx : numpy.ndarray, indices of the vertices
-            of the region to be plotted
-        """
-        # count how many vertices belong to the given parcellation in each face
-        verts_per_face = np.isin(self._faces, parc_idx).sum(axis=1)
-
-        # test if parcellation forms regions
-        if np.all(verts_per_face < 2):
-            raise ValueError("Vertices in parcellation do not form region.")
-
-        vertices_on_edge = np.intersect1d(
-            np.unique(self._faces[verts_per_face == 2]), parc_idx
-        )
-        faces_outside_edge = np.isin(self._faces, vertices_on_edge).sum(axis=1)
-
-        return np.logical_and(faces_outside_edge > 0, verts_per_face < 3)
-
```

Comment:
This function is the same as `_get_faces_on_edge` in `surf_plotting`. I've moved that one to `nilearn.plotting.surface._utils` and I'm using that one in `PlotlySurfaceFigure`.